### PR TITLE
docs(bodiless-layouts): Content editor can create Library Item (#1194)

### LINKED ISSUE #1194: docs(bodiless-layouts): Content editor can create Library Item
## User Story
AS AN end user of the Bodiless documentation,
I WANT to understand how to use the Content Library,
SO THAT I can add an item to the Content Library using the Editor Interface.

## Acceptance Criteria
1. The Documentation is updated to include how a Site Builder can configure a FlowContainer to use the Content Library.
2. The Documentation is updated to include how a Content Editor can use the Content Library (add items to it, and use items from it).

---

**Related Issue:** #839 Content editor can create Library Item

### PR DESCRIPTION
**Pull request by misterjones**
## Changes
- Update documentation to include:
  - How a Site Builder can configure a FlowContainer to use the Content
    Library.
  - How a Content Editor can use the Content Library (add items to it,
    and use items from it).

## Related Issues
Fixes #1194

### CODE REVIEW

File: packages/bodiless-layouts/README.md [hunk 1]
```diff
@@ -26,6 +29,8 @@ components. You can filter the components by:
 * Using search facets to filter out components that do not match the selection 
 (you can undo this by clicking the "select all" checkbox at the top).
 * Using the search box field to search across all of the component titles.
+* If the Content Library has been enabled in the current Flow Container, you can select the "Content
```
Comment (dewen) [inline packages/bodiless-layouts/README.md:32]: If 
- the Content Library has been enabled in the current Flow Container
- and there are existing content data available in library
...

File: packages/bodiless-documentation/doc/ContentEditorUserGuide/README.md [hunk 2]
```diff
@@ -292,3 +292,11 @@ the links below to read more about working with the individual components.
 * Robots.txt 
 * [Metatags](../../../Development/Guides/BuildingSites/ComponentsStarterKit?id=metadata-component) 
 
+### Content Library
+
+Using the _Content Library_, you can save components that you've created, along with their embedded
+content, allowing you to reuse them elsewhere on your site. The Content Library can be configured
+for use in [Flow Containers](/Components/FlowContainer); once enabled, you will be able to:
+
+* Save components in your Flow Container to the Content Library; and
```
[the remark this answers is not in the document]
Comment (misterjones) [reply packages/bodiless-documentation/doc/ContentEditorUserGuide/README.md]: Linked. ✅

File: packages/bodiless-documentation/doc/ContentEditorUserGuide/README.md [hunk 3]
```diff
@@ -292,3 +292,11 @@ the links below to read more about working with the individual components.
...
+* Add components saved in the Content Library to your Flow Container.
```
[the remark this answers is not in the document]
Comment (misterjones) [reply packages/bodiless-documentation/doc/ContentEditorUserGuide/README.md:303]: Linked. ✅

File: packages/bodiless-layouts/README.md (hunk 1 above)
Comment (misterjones) [reply packages/bodiless-layouts/README.md]: Updated with clarification. ✅

File: packages/bodiless-layouts/README.md [hunk 4]
```diff
@@ -450,3 +472,84 @@ A few things to note when using the component selector independently:
...
+```
+
+```tsx
+import { withLibraryComponents } from '@bodiless/layouts';
+import { FlowContainer } from '@bodiless/layouts-ui';
+import { withLibraryFlowContainerVariations } from './withFlowContainerVariations';
+import asDefaultFlowContainer from './asDefaultFlowContainer';
+
+const FlowContainerDefaultWithContentLibrary = asToken(
+  // withLibraryComponents should be applied before any other HOC designs.
```
[the remark this answers is not in the document]
Comment (misterjones) [reply packages/bodiless-layouts/README.md:546]: Added a note before this particular code block. ✅ 

I apologize if this conflicts with your updates to the placeholder code.

[the remark this answers is not in the document]
Comment (dewen) [reply packages/bodiless-layouts/README.md:546]: no worries

**Review by dewen — APPROVED** [review]

**Pull request merged**